portal/src/pages/components/accordion.tsx

```diff
@@ -0,0 +1,33 @@
+// @ts-ignore
+import ButtonExampleCode from "!raw-loader!../../examples/AccordionExample";
+// @ts-ignore
+import AccordionMarkdown from "!raw-loader!@fremtind/jkl-accordion/README.md";
+import React from "react";
+import ReactMarkdown from "react-markdown";
+import { AccordionExample } from "../../examples/AccordionExample";
+import Layout from "../../components/Layout";
+import { PrismHighlightedCode } from "../../components/PrismHighlightedCode";
+import { renderer } from "../../presentation/markdownRenderer";
+import { Accordion, AccordionItem } from "@fremtind/jkl-accordion-react";
+
+export default function Button() {
+    return (
+        <Layout>
+            <h2 className="jkl-h1 portal-content__title">Trekkspillliste</h2>
```

Comment:
Can't this be a ExampleTemplate?

```
const ExampleTemplate = (title, Example, ExampleCode, Markdown)  => (
<Layout>
            <h2 className="jkl-h1 portal-content__title">{title}</h2>
            <div>
                <div className="portal-content__example">
                    <Example />
                </div>

                <Accordion>
                    <AccordionItem title="Kode">
                        <PrismHighlightedCode>{ExampleCode}</PrismHighlightedCode>
                    </AccordionItem>
                </Accordion>
            </div>
            <div style={{ margin: "2rem 0" }}>
                <ReactMarkdown renderers={renderer} source={Markdown} />
            </div>
        </Layout>);
```

Reply:
The reason it is not a template is in case you want to add multiple usage examples and/or developer specific documentation. We need to look closer at structure either way. Right now it feels fine to just copy/paste something simple to get _some_ content out, and then we can worry about optimizing it when we see what our needs are..

Reply:
What if Example and ExampleCode were ReactNodes or array of ReactNodes? And if array, render them all? But yeah, there is probably other things that is more important at this stage.